server/jetstream.go

```diff
@@ -207,11 +208,14 @@ func (s *Server) enableJetStream(cfg JetStreamConfig) error {
 	s.Noticef("  Max Memory:      %s", friendlyBytes(cfg.MaxMemory))
 	s.Noticef("  Max Storage:     %s", friendlyBytes(cfg.MaxStore))
 	s.Noticef("  Store Directory: \"%s\"", cfg.StoreDir)
+	if cfg.Domain != _EMPTY_ {
+		s.Noticef("  Enable Domain:   %s", cfg.Domain)
```

Comment:
I guess this can just be `Domain` though?

Reply:
Sure.